docs/providers.rst

```diff
@@ -363,7 +363,6 @@ AsyncHTTPProvider
     The only documented exceptions to this are:
```

Comment:
Maybe get rid of this exceptions line too? I don't think there are any exceptions listed after it now that the ENS line is gone.

Reply:
Nice catch! Removed!